recipes/detectron2/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "detectron2" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fd874aa47414d3de2bc9e4bb405090aa2f3a14e17903ea65afcb6ed106d804ba
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pytorch
+  run:
+    - python
```

Comment:
Also, can you please specify the min versions for some of the dependencies as listed here, https://github.com/facebookresearch/detectron2/blob/1f6ebff69b79f93d69e59eca9c2e84a9f03d850e/setup.py#L130 ? I just want to avoid the possibility of the package being installed with lower versions and causing issues.